chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -134,8 +134,10 @@ sync_time() {
 
 <% if @target_platform_version.to_f < 12.0 %>
     SNTP_OPTS="-P no -r"
-<% else %>
+<% elsif @target_platform_version.to_f == 12.0 %>
```

Comment:
then I'd prefer < 12.09 exactly to avoid float comparison issues. 12.0 can be represented exactly as a float